src/app/core/footer/footer.component.css

```diff
@@ -9,6 +9,7 @@ footer {
   height: var(--footer-height);
   display: flex;
   flex-direction: column;
+  z-index: 1000;
```

Comment:
...why 1000? Isn't the point to make it not selectable?

Reply:
Lol why does the footer need a `z-index` at all?

Reply:
We need the footer to be on top of all other elements on the site